src/pages/getting-started/app-guides/helpdesk-widgets/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+weight: 80
```

Comment:
@oliwiapolec if you could please look at it :) It is the equivalent of Agent App Widgets BB in LCH. It has a new name so it was decided it will be added as a separate one.

Reply:
I see it's on the bottom of the list - maybe we could experiment with the weight so we could put it under the Agent App Widgets as it's corresponding to that? 👀

Reply:
🤔 hmm I see it under Chat Webhooks - unless we are talking about something different
<img width="263" alt="Screenshot 2022-09-15 at 10 56 51" src="https://user-images.githubusercontent.com/102953526/190361499-2eb872dc-72f5-4ca5-9524-d10519735b9c.png">

Reply:
Yeah, I meant it's rather in the bottom than on the top >D All in all my thought was to put it under Agent App Widgets as it's directly related to Widgets, so it's a "pair" XD

Reply:
Ohhh okay hahaha xd when talking with Ola we agreed that the alphabetical order should be left here as the list of which BBs belong to which product will be visible in the overview, and when someone wants to look for a specific BB, they will basically search for its name :)